isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -1411,4 +1363,231 @@ QString BundleObservation::formatBundleOutputString(bool errorPropagation, bool
+                                                    &coeffImage(0, index),
+                                                    &coeffImage(1, index));
+        index++;
+      }
+
+      // Add the partial for the y coordinate of the position
+      for (int cameraCoef = 0; cameraCoef < numCamPositionCoefficients; cameraCoef++) {
+        measureCamera->GroundMap()->GetdXYdPosition(SpicePosition::WRT_Y, cameraCoef,
+                                                    &coeffImage(0, index),
+                                                    &coeffImage(1, index));
+        index++;
+      }
+
+      // Add the partial for the z coordinate of the position
+      for (int cameraCoef = 0; cameraCoef < numCamPositionCoefficients; cameraCoef++) {
+        measureCamera->GroundMap()->GetdXYdPosition(SpicePosition::WRT_Z, cameraCoef,
+                                                    &coeffImage(0, index),
+                                                    &coeffImage(1, index));
+        index++;
+      }
+    }
+
+    // Partials for RA, DEC, twist
+    if (observationSolveSettings->instrumentPointingSolveOption() !=
+        BundleObservationSolveSettings::NoPointingFactors) {
+
+      int numCamAngleCoefficients =
+          observationSolveSettings->numberCameraAngleCoefficientsSolved();
+
+      // Add the partials for ra
+      for (int cameraCoef = 0; cameraCoef < numCamAngleCoefficients; cameraCoef++) {
+        measureCamera->GroundMap()->GetdXYdOrientation(SpiceRotation::WRT_RightAscension,
+                                                       cameraCoef, &coeffImage(0, index),
+                                                       &coeffImage(1, index));
+        index++;
+      }
+
+      // Add the partials for dec
+      for (int cameraCoef = 0; cameraCoef < numCamAngleCoefficients; cameraCoef++) {
+        measureCamera->GroundMap()->GetdXYdOrientation(SpiceRotation::WRT_Declination,
+                                                       cameraCoef, &coeffImage(0, index),
+                                                       &coeffImage(1, index));
+        index++;
+      }
+
+      // Add the partial for twist if necessary
+      if (observationSolveSettings->solveTwist()) {
+        for (int cameraCoef = 0; cameraCoef < numCamAngleCoefficients; cameraCoef++) {
+          measureCamera->GroundMap()->GetdXYdOrientation(SpiceRotation::WRT_Twist,
+                                                         cameraCoef, &coeffImage(0, index),
+                                                         &coeffImage(1, index));
+          index++;
+        }
+      }
+    }
+
+    return true;
+  }
+
+
+  bool BundleObservation::computePoint3DPartials(matrix<double> &coeffPoint3D, BundleMeasure &measure, SurfacePoint::CoordinateType coordType) {
+    coeffPoint3D.clear();
+    Camera *measureCamera = measure.camera();
+    BundleControlPoint* point = measure.parentControlPoint();
```

Comment:
For this function I think passing the measure makes sense.

Reply:
Agreed.